server/policy_constants.go

```diff
@@ -60,6 +60,14 @@ var (
 	RevokedCiscoCert []byte
 )
 
+// Known Privacy CA certs.
+var (
+	//go:embed ca-certs/tpm_ek_root_1.cer
+	GceEKRootCA []byte
+	//go:embed ca-certs/tpm_ek_intermediate_2.crt
+	GceEKIntermediateCA2 []byte
+)
+
```

Comment:
Nit, these should both be `*x509.CertPool`s (created from the embedded data), and they should be named something like: `GceEKRoots` and `GceEKIntermediates`